actuator-http-metrics/src/main/java/sample/camel/Application.java

```diff
@@ -27,6 +29,11 @@
 @SpringBootApplication
 public class Application {
 
+    @Bean
+    public MicrometerRoutePolicyFactory micrometerRoutePolicyFactory() {
+        return new MicrometerRoutePolicyFactory();
+    }
```

Comment:
is it necessary or it is enough the [autoconfiguration](https://github.com/jboss-fuse/camel-spring-boot/blob/camel-spring-boot-4.0.0-branch/components-starter/camel-micrometer-starter/src/main/java/org/apache/camel/component/micrometer/springboot/metrics/CamelMetricsAutoConfiguration.java#L46-L48) ? I know it is different I didn't test it

Reply:
I think this may have been necessary for Camel 3, but I've tried with/without it and it doesn't look like it's necessary for Camel 4.    I've removed that method in all of the sample apps.